material_studies/rally/pubspec.yaml

```diff
@@ -20,20 +20,20 @@ flutter:
   assets:
     - assets/logo.png
     - assets/thumb.png
-#  fonts:
-#    - family: Roboto Condensed
-#      fonts:
-#        - asset: fonts/RobotoCondensed-Light.ttf
-#          weight: 400
-#        - asset: fonts/RobotoCondensed-Regular.ttf
-#          weight: 500
-#        - asset: fonts/RobotoCondensed-Bold.ttf
-#          weight: 700
-#    - family: Eczar
-#      fonts:
-#        - asset: fonts/Eczar-Regular.ttf
-#          weight: 400
-#        - asset: fonts/Eczar-SemiBold.ttf
-#          weight: 600
-#        - asset: fonts/Eczar-Bold.ttf
-#          weight: 700
+  fonts:
+    - family: Roboto Condensed
+      fonts:
+        - asset: fonts/RobotoCondensed-Light.ttf
+          weight: 400
+        - asset: fonts/RobotoCondensed-Regular.ttf
+          weight: 500
+        - asset: fonts/RobotoCondensed-Bold.ttf
+          weight: 700
+    - family: Eczar
+      fonts:
+        - asset: fonts/Eczar-Regular.ttf
```

Comment:
Is this a Google font? If not, be sure to include the appropriate license.

Reply:
Are there any examples/instructions of how to add the appropriate license? Thanks!

Reply:
Looks like it's an OTF font:

http://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL_web

I believe there are other material samples that include that license. @clocksmith and @willlarche would know for sure.